templates/firefox-ios.xliff

```diff
@@ -138,8 +138,8 @@
         <source>Passcode</source>
         <note>Label for the Passcode item in Settings</note>
       </trans-unit>
-      <trans-unit id="Passcodes didn't match. Try again.">
-        <source>Passcodes didn't match. Try again.</source>
+      <trans-unit id="Passcodes didn’t match. Try again.">
+        <source>Passcodes didn’t match. Try again.</source>
```

Comment:
This is not good, we should not be invalidating all these translations because English started using proper apostrophes. Each locale has its own rules regarding quotes and apostrophes.

We need a temporary script that keeps these existing translation, if available, for all this kind of changes.

Reply:
The other option is to convert these strings (and the rest in that group) to use proper identifiers.

Reply:
It wouldn't solve the problem, the script is built to discard changes to strings without new IDs.